docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.5"
+services:
+  dbnode01:
+    expose:
+      - "9000-9004"
+    ports:
+      - "0.0.0.0:9000-9004:9000-9004"
+      - "0.0.0.0:7201:7201"
+      - "0.0.0.0:2379-2380:2379-2380"
+    networks:
+      - backend
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: m3db_dbnode01:latest
```

Comment:
Should we update this to our latest quay image?

Reply:
Are the quay images being published regularly?

Reply:
Once we start tagging releases we'll have images on quay whose tags correspond to the git tag, and `quay.io/m3db/m3db:latest` will always point at the latest tagged release. In the mean time we're pushing builds periodically as we fix bugs on master.

I don't think we'll want to push a new sha-tagged image for every commit that lands on master. Or if we do, I think we'll want to clean up old ones regularly since untagged releases should be considered dev builds.

Reply:
Never mind, didn't realize docker compose will tag this locally